### PR TITLE
add phone image prop

### PR DESCRIPTION
**Pull request by IncognitaDev**
#### What problem is this solving?

This prop alows to use other image for phone size.  

#### How to test it?

just add the block in your theme and pass phoneImage prop

[Workspace](https://sudambeef--agenciaeplus.myvtex.com/)

#### Screenshots or example usage:

desktop
image

mobile / tablet
image

phone
image

obs: is just test images.

#### Describe alternatives you've considered, if any.

use Responsive layout can be usefull but this is more work to the client.

### CODE REVIEW

File: react/modules/imageAsList.tsx [hunk 1]
```diff
@@ -3,9 +3,19 @@ import React from 'react'
 import Image from '../Image'
 import type { ImagesSchema } from '../ImageTypes'
 
+const getResponsiveImage = (device: string, image: string, mobileImage: string,phoneImage: string) => {
```
Comment (igorbrasileiro) [inline react/modules/imageAsList.tsx]:
```suggestion
interface ResponsiveImageParams {
  device: string
  image: string
  mobileImage: string
  phoneImage: string
}
const getResponsiveImage = ({ device, image, mobileImage, phoneImage }: ResponsiveImageParams) => {
```

File: store/interfaces.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
   "image-new": {
     "component": "Image",
     "content": {
-      "$ref": "app:vtex.store-image#/definitions/Image"
+      "$ref": "app:sudambeef.store-image#/definitions/Image"
```
Comment (igorbrasileiro) [inline store/interfaces.json]: Fix this interface.

File: store/interfaces.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "content": {
       "properties": {
         "images": {
-          "$ref": "app:vtex.store-image#/definitions/Images"
+          "$ref": "app:sudambeef.store-image#/definitions/Images"
```
Comment (igorbrasileiro) [inline store/interfaces.json]: Add `vtex` back.

File: store/interfaces.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
...
```
Comment (igorbrasileiro) [inline store/interfaces.json]: Add `vtex` back.

File: store/interfaces.json [hunk 5]
```diff
@@ -2,15 +2,15 @@
...
+      "$ref": "app:sudambeef.store-image#/definitions/Image"
     }
   },
   "image-slider": {
     "component": "ImageSlider",
     "content": {
       "properties": {
         "images": {
-          "$ref": "app:vtex.store-image#/definitions/Images"
+          "$ref": "app:sudambeef.store-image#/definitions/Images"
```
Comment (igorbrasileiro) [inline store/interfaces.json]: Add `vtex` back.

File: store/interfaces.json (hunk 2 above)
Comment (igorbrasileiro) [inline store/interfaces.json]: Add `vtex` back.

File: store/contentSchemas.json [hunk 6]
```diff
@@ -97,6 +97,14 @@
               "ui:widget": "image-uploader"
             }
           },
+          "phoneImage": {
+            "$ref": "app:vtex.native-types#/definitions/url",
+            "default": "",
+            "title": "admin/editor.image-list.images.mobileImage.title",
```
Comment (igorbrasileiro) [inline store/contentSchemas.json]: This isn't a mobile image. Can you add a new entry for this message?

**Review by igorbrasileiro — CHANGES_REQUESTED** [review]
Great job, overall. 
Can you lint all files? Go to the root folder, install the npm packages, then run `yarn lint`.
Also, I've left some code suggestions.

File: messages/bg-BG.json [hunk 7]
```diff
@@ -9,6 +9,7 @@
   "admin/editor.image-list.images.title": "Изображения",
   "admin/editor.image-list.images.image.title": "Изображение",
   "admin/editor.image-list.images.mobileImage.title": "Изображение за мобилно устройство",
+  "admin/editor.image-list.images.phoneImage.title": "изображение на мобилен телефон",
```
Comment (camilavcoutinho) [inline messages/bg-BG.json]:
```suggestion
  "admin/editor.image-list.images.phoneImage.title": "Изображение за телефон",
```

File: messages/it-IT.json [hunk 8]
```diff
@@ -9,6 +9,7 @@
   "admin/editor.image-list.images.title": "Immagini",
   "admin/editor.image-list.images.image.title": "Immagine",
   "admin/editor.image-list.images.mobileImage.title": "Immagine per dispositivi mobili",
+  "admin/editor.image-list.images.phoneImage.title": "Immagine per telefonino",
```
Comment (camilavcoutinho) [inline messages/it-IT.json]:
```suggestion
  "admin/editor.image-list.images.phoneImage.title": "Immagine per cellulari",
```

File: messages/ja-JP.json [hunk 9]
```diff
@@ -9,6 +9,7 @@
   "admin/editor.image-list.images.title": "画像",
   "admin/editor.image-list.images.image.title": "画像",
   "admin/editor.image-list.images.mobileImage.title": "モバイル用画像",
+  "admin/editor.image-list.images.phoneImage.title": "携帯電話用画像",
```
Comment (camilavcoutinho) [inline messages/ja-JP.json]:
```suggestion
  "admin/editor.image-list.images.phoneImage.title": "電話用画像",
```

File: messages/ko-KR.json [hunk 10]
```diff
@@ -9,6 +9,7 @@
   "admin/editor.image-list.images.title": "이미지",
   "admin/editor.image-list.images.image.title": "이미지",
   "admin/editor.image-list.images.mobileImage.title": "모바일용 이미지",
+  "admin/editor.image-list.images.phoneImage.title": "휴대폰용 이미지",
```
Comment (camilavcoutinho) [inline messages/ko-KR.json]:
```suggestion
  "admin/editor.image-list.images.phoneImage.title": "휴대폰의 이미지",
```

File: messages/ro-RO.json [hunk 11]
```diff
@@ -9,6 +9,7 @@
   "admin/editor.image-list.images.title": "Imagini",
   "admin/editor.image-list.images.image.title": "Imagine",
   "admin/editor.image-list.images.mobileImage.title": "Imagini pentru mobil",
+  "admin/editor.image-list.images.phoneImage.title": "Imagini pentru telefon mobil",
```
Comment (camilavcoutinho) [inline messages/ro-RO.json]:
```suggestion
  "admin/editor.image-list.images.phoneImage.title": "Imagine pentru telefon mobil",
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

**Review by camilavcoutinho — APPROVED** [review]

**Review by igorbrasileiro — APPROVED** [review]

**Comment by pachecovit** [comment]
Aprovem o PR do cara! Essa feature está sendo de grande importancia para a comunidade VTEX hehehhuhe

**Pull request closed without merging**